cyflash/bootload.py

```diff
@@ -8,9 +8,10 @@
 
 from builtins import input
 
-from . import cyacd
-from . import protocol
-
+#from . import cyacd
```

Comment:
remove this if it shouldn't be there anymore

Reply:
i fix this after i understand a better solution...

Reply:
restored.